acceptance/tests/direct_puppet/cached_catalog_remediate_local_drift.rb

```diff
@@ -117,8 +117,7 @@
 MANIFEST
 
       step "Run agent again using --use_cached_catalog and ensure content from the first code_id is used"
-      on(agent, puppet("agent", "--no-usecacheonfailure", "--no-splay", "--show_diff", "--no-daemonize", 
-                       "--verbose", "--onetime", "--detailed-exitcodes", "--use_cached_catalog", "--server #{master}"), :acceptable_exit_codes => [0,2])
+      on(agent, puppet("agent", "-t", "--use_cached_catalog", "--server #{master}"), :acceptable_exit_codes => [0,2])
```

Comment:
How did this test ever pass if adding `--use_cached_catalog` did not override what was in `--test`...?